src/features/Nav/Nav.tsx

```diff
@@ -15,8 +15,10 @@ export const Nav = styled.nav<INavProps>`
   display: flex;
   justify-content: space-between;
   background: ${(props) => props.theme.colors.white};
-  border-bottom: ${(props) => props.borderThickness || '2px'} solid
+  border-bottom: ${(props) => (props.hasBorder ? '2px solid' : '0px')}
```

Comment:
Border animation is a little funky on chrome/windows 10 (think it's because it's growing from 0px to 2px, causing it to look like animation is stuttering).

Should be fixed by:
```
background: ${(props) => props.hasBorder ? props.theme.colors.white : 'transparent'};
border-bottom: 2px solid transparent;
border-color: ${(props) => props.hasBorder ? props.theme.colors.black5 : 'transparent'};
transition: 0.25s all ease-in;
```

Reply:
good catch! thanks